apps/calendar/style/settings.css

```diff
@@ -32,14 +32,6 @@ body[data-path^='/settings'].pending-operation #progress-indicator {
   background: transparent;
 }
 
-/*
-The text in the back button is just for accessibility concerns,
-not actually shown since the top header should be transparent.
-*/
-#settings .settings-back {
```

Comment:
Yeah that's the type of thing i was hoping to remove, but this is related to a different button. It can be a follow up if you'd like to open a bug, but we should not remove this stuff in this PR